docs/eks-installation.md

```diff
@@ -0,0 +1,34 @@
+# Deploying Antrea in AWS EKS
```

Comment:
We might add a link to getting-started.md to this doc. @antoninbas

Reply:
sounds good to me. Next to the "Deploying Antrea in Kind" section, we can have a "Deploying Antrea in Cloud Managed K8s Services" section, to indicate that for now we have EKS support and point to this doc.

Reply:
may be another PR?